aeron-driver/src/main/c/aeron_driver_conductor.c

```diff
@@ -4495,20 +4495,21 @@ void aeron_driver_conductor_on_create_publication_image(void *clientd, void *ite
     {
         AERON_APPEND_ERR("%s", "");
         aeron_driver_conductor_log_error(conductor);
-        return;
+        goto error_cleanup;
     }
 
     if (!aeron_driver_conductor_has_network_subscription_interest(
         conductor, endpoint, command->stream_id, command->session_id))
     {
-        return;
+        AERON_SET_ERR(AERON_ERROR_CODE_UNKNOWN_SUBSCRIPTION, "Image doesnt have network subscription interest", "");
```

Comment:
Admittedly I'm not sure how this situation can occur, or if it is an error case really?

Reply:
It isn't really an error condition.  This is to short circuit/no-op a case where the driver receives a setup for a stream that doesn't have any subscriptions.  I suspect that this code may even be redundant as the only code path that generates this is when we have a subscription that is awaiting setup, in which case it should have network subscription interest.

Reply:
Yeah I wasn't sure if it was possible to hit this with a small gap between subscription creation and the unsubscribe. Potentially if the conductor queue is a bit backed up?

Reply:
I've removed the error from here, but still clear the state in the receiver packet dispatcher